internal/rbd/rbd_attach.go

```diff
@@ -238,7 +250,26 @@ func appendDeviceTypeAndOptions(cmdArgs []string, isNbd, isThick bool, userOptio
 	return cmdArgs
 }
 
-func createPath(ctx context.Context, volOpt *rbdVolume, cr *util.Credentials) (string, error) {
+// appendRbdNbdCliOptions append manditory options and convert list of useroptions
```

Comment:
nit; manditory/mandatory

Reply:
wonder, how the codespell missed it? does it ignore the comments?